vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -71,15 +70,15 @@ public RedirectView createOrUpdateExhibition(HttpServletRequest request,
         ISpace startSpace = spaceManager.getSpace(spaceID);
 
         Exhibition exhibition;
-        if(exhibitID==null || exhibitID.isEmpty()) {
+        if (exhibitID == null || exhibitID.isEmpty()) {
             exhibition = (Exhibition) exhibitFactory.createExhibition();
```

Comment:
wouldn't you want the exhibition to be created with a preview id here as well?

Reply:
also, what's the difference between `exhibitFactory.createExhibition()` and `exhibitManager.createExhibitionWithPreviewId()`? There should just be one way to create an exhibition and it should always have a preview id.